src/lib/create.c

```diff
@@ -301,7 +301,7 @@ hash_bn(pgp_hash_t *hash, BIGNUM *bignum)
 static int
 hash_key_material(const pgp_seckey_t *key, uint8_t *result)
 {
-    pgp_hash_t hash;
+    pgp_hash_t hash = {0};
```

Comment:
I think that `memset(&hash, 0, sizeof(pgp_hash_t)` better because this way doesn't depend on `pgp_hash_t` structure size/fields.